app/constants/calls.ts

```diff
@@ -32,6 +32,11 @@ const REACTION_TIMEOUT = 10000;
 const REACTION_LIMIT = 20;
 const CALL_QUALITY_RESET_MS = toMilliseconds({minutes: 1});
 const CAPTION_TIMEOUT = 5000;
+const RING_LENGTH = 30000;
+const RINGTONE_DEFAULT = 'Calm';
+
+// 30 seconds of vibration (there is no loop setting)
```

Comment:
I see, it would be nice to find a way to have a loop, although 30 seconds should be enough

Reply:
Yep, no loop available. But 30 seconds is the max ring duration anyway.